tests/functional/test_useragent.py

```diff
@@ -10,7 +10,7 @@ def assert_expected_user_agent(result, service, operation):
     assert ' md/arch#' in ua_string
     assert ' md/prompt#off' in ua_string
     assert f' md/command#{service}.{operation}' in ua_string
-    assert ' ua/2.0 ' in ua_string
+    assert ' ua/2.1 ' in ua_string
```

Comment:
Not a blocker, but might be nice to assert the `m/C` for the `aws s3 ls` case below while we're here.

Reply:
Addressed in https://github.com/aws/aws-cli/pull/9333/commits/edb58d5e13871c45da53eab79a9e8d88ffc48281